docs/intro.rst

```diff
@@ -0,0 +1,111 @@
+Camus
+================
+
+Camus is a simple MapReduce job developed by LinkedIn to load data from Kafka into HDFS.
+It is capable of incrementally copying data form Kafka into HDFS such that
+every run of the MapReduce job picks up from where the previous run left off.
+At LinkedIn, Camus is used to load billions of messages per day from Kafka into HDFS.
+Confluent's version of Camus integrates with Confluent's schema registry which automatically
+validates the data integrity as it is loaded and ensure compatibility of data stored in HDFS
+as schemas are evolved.
+
+Quickstart
+----------
+
+Assuming you have already installed Confluent Platform. You'll need to specify the location
+of Camus's jar file when running the MapReduce job. If you install Camus via zip/tgz archive,
+you can find Camus's jar under ``share/java/camus/``. If you install Camus via rpm or deb,
+the Camus's jar is under ``/usr/share/java/camus/``.
```

Comment:
Preemptive note for @nehanarkhede: I'm going to clean up these two sections since they also need some other changes copied from the kafka-rest setup, so I think the current form is fine for merging this patch.